### DISCUSSION TITLE
show scale of the map

### DISCUSSION THREAD

**Discussion by voxspox** [organicmaps/organicmaps#4289] [Ideas]
Hi, maybe I didn't found the enable switch... but I'm missing to see the scale of the map on the map (this little bar thing which let me understand how much millimeters on the map are how much meters in real world)

**Comment by Misalf-git** [comment]
No switch. Just wait for the "Maps data © OpenStreetMap" message to disappear at the lower left.
![Omaps.png](https://user-images.githubusercontent.com/72605561/212675573-49b3fa1a-038a-49cb-a396-a0812dec0d79.png)
